packages/aragon-wrapper/src/utils/index.js

```diff
@@ -16,3 +16,20 @@ export function makeProxy (address, interfaceName, web3, initializationBlock) {
 export function makeProxyFromABI (address, abi, web3, initializationBlock) {
   return new Proxy(address, abi, web3, initializationBlock)
 }
+
+export const getRecommendedGasLimit = async ({ web3, estimatedGasLimit }) => {
+
+  const latestBlock = await web3.eth.getBlock('latest')
+  const latestBlockGasLimit = latestBlock.gasLimit;
+
+  const upperGasLimit = latestBlockGasLimit*0.95
```

Comment:
@izqui I went with `0.95` now as per https://github.com/aragon/aragon.js/issues/150#issuecomment-420990894

Let me know if this needs adjustment.